requirements-dev.txt

```diff
@@ -1,6 +1,7 @@
-git+https://github.com/spacetelescope/asdf#egg=asdf
-git+https://github.com/astropy/astropy#egg=astropy
-git+https://github.com/spacetelescope/crds#egg=crds
-git+https://github.com/spacetelescope/gwcs#egg=gwcs
-git+https://github.com/astropy/photutils#egg=photutils
-git+https://github.com/spacetelescope/tweakwcs#egg=tweakwcs
+git+https://github.com/spacetelescope/asdf
```

Comment:
Can't make an omelette without breaking a few packages

Reply:
Ha!  The `egg=` portion here is not necessary, as the package name and egg name are the same.